scripts/Muon/GUI/FrequencyDomainAnalysis/MaxEnt/maxent_presenter.py

```diff
@@ -23,6 +23,10 @@ def __init__(self, view, alg, load):
         self.view.cancelSignal.connect(self.cancel)
         self.view.phaseSignal.connect(self.handlePhase)
 
+    @property
+    def widget(self):
+        return self.view
+
     # functions
     def getWorkspaceNames(self):
```

Comment:
This functions seems to do more than getting workspace names.

Reply:
it makes sure that the user does not use less data points then are available, since this is view we update it with adding the workspaces to the view